Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst

```diff
@@ -272,8 +272,12 @@ First we extend the database schema by adding the following to the file
 .. tip::
 
     Do a database compare in the :guilabel:`Admin Tools > Maintenance` module
-    after changing the database schema (system maintainers only).
+    after changing the database schema (system maintainers only). Or call the
+    console command:
 
+    ..  code-block:: bash
```

Comment:
Again me ;-) We should use tabs for distinction of Composer-based and legacy installation, like here: https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/ApiOverview/CommandControllers/Index.html

Reply:
With the new parser I intentend to introduce a directive for that. Right now it is just a hussle...